tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -191,9 +195,10 @@ namespace tensorflow {
 //        B = MaxPoolGrad(X, A, Y)
 //
 //        We will transform this graph to propagate workspace as:
+//        (with contiguous ordering)
```

Comment:
Yes, I did add it - https://github.com/tensorflow/tensorflow/pull/8968/commits/67f9925ef9ceed02892c200a3122092ab497943a#diff-2359860d8d805c348b1e2af4598b43dd